docs/src/pages/docs/hooks/use-cda-time-series-group.jsx

```diff
@@ -0,0 +1,126 @@
+import { H3, Text } from "@usace/groundwork";
+import ParamsTable from "../../components/params-table";
+import CdaParamsTable from "../../components/cda-params-table";
+import QueryClientWarning from "../../../components/QueryClientWarning";
+import { useCdaTimeSeriesGroup } from "@usace-watermanagement/groundwork-water";
+import { Code as CodeBlock } from "../../components/code";
+import { Code } from "@usace/groundwork";
+import DocsPage from "../_docs-wrapper";
+import Divider from "../../components/divider";
+
+const cdaParams = [
+  {
+    name: "groupId",
+    type: "string",
+    required: true,
+    desc: "Specifies the timeseries group whose data is to be included in the response.",
+  },
+  {
+    name: "office",
+    type: "string",
+    required: true,
+    desc: "Specifies the owning office of the timeseries group whose data is to be included in the response.",
+  },
+  {
+    name: "categoryId",
+    type: "string",
+    required: true,
+    desc: "Specifies the category containing the timeseries group whose data is to be included in the response.",
+  },
+];
+
+const TimeSeriesGroupList = () => {
+  const { data, isPending, isError } = useCdaTimeSeriesGroup({
+    cdaParams: {
+      groupId: "USGS TS Data Acquisition",
+      office: "CWMS",
+      categoryId: "Data Acquisition",
+    },
+  });
+
+  if (isPending) return <span>Loading time series group data...</span>;
+  if (isError) return <span>Time series group error!</span>;
+
+  return (
+    <div>
+      <H3>USGS Data Acquisition</H3>
+      <ul>
+        {data.assignedTimeSeries
+          .slice(0, 5) // Get the first two key-value pairs
```

Comment:
First 2? Wouldn't slice give the first 5?

Figure it's just a forgotten comment